presto-server-rpm/src/main/rpm/preinstall

```diff
@@ -22,7 +22,11 @@ check_if_correct_java_version() {
 # candidate for JAVA_HOME).
   JAVA_VERSION=$(java_version "$1")
   JAVA_UPDATE=$(echo $JAVA_VERSION | cut -d'_' -f2)
-  if [[ ("$JAVA_VERSION" > "1.8") && ($JAVA_UPDATE -ge 161) ]]; then
+  JAVA_MAJOR=$(echo $JAVA_VERSION | cut -d'.' -f1)
+  if [[ ("$JAVA_MAJOR" -ge "11") ]]; then
```

Comment:
what about other versions? Can we make the mechanism generic?

Reply:
This is quite generic for example `12 > 11`. However then env variable `JAVA11_HOME` has unfortunate name.